packages/connectivity/src/scp-cf/destination/http-proxy-util.ts

```diff
@@ -1,8 +1,12 @@
 import { AgentOptions } from 'https';
-import { URL } from 'url';
+import { URL } from 'node:url';
```

Comment:
What's the difference between the two imports?

Reply:
It is more a semantic information.